src/Core/PotreeNode.js

```diff
@@ -80,28 +80,18 @@ class PointCloudNode {
         throw new Error(`Cannot find node with name '${name}'`);
     }
 
-    octreeUrl() {
-        return `${this.baseurl}/r${this.name}.${this.layer.source.extensionOctree}`;
-    }
-
-    nodeUrl() {
```

Comment:
Why don't you keep those methods ?

Reply:
Because they were only used once.

Reply:
I add a new commit:
The `PotreeNode` have a `load` function and `url getter`.
This fits better with the future architecture of `itowns`. Where the `command` will perform `node load`.